CHANGELOG.rst

```diff
@@ -5,6 +5,20 @@ Junipernetworks Junos Collection Release Notes
 .. contents:: Topics
 
 
+v2.5.0
+======
+
+Minor Changes
+-------------
```

Comment:
These should be trivial changes. Not minor. Since these are not user facing ones.

Reply:
And also these should not be a part of changelog.